vendor/maplibre-override.css

```
@@ -0,0 +1,31 @@
+.maplibregl-ctrl-top-right .maplibregl-ctrl {
+  margin: 60px 25px 0 0;
+  border-radius: 0;
+  background: transparent;
+}
+
+.maplibregl-ctrl-group:not(:empty) {
+  box-shadow: none;
+}
+
+.maplibregl-ctrl button:not(:disabled):hover {
+  background-color: var(--hover-main);
+}
+
+.maplibregl-ctrl button {
+  background-color: var(--main);
+  color: white;
+  width: 30px;
+  height: 30px;
+}
+
+.theme-light.maplibregl-ctrl button {
+  background-color: white;
+  color: var(--main);
+  stroke: none;
+}
```

Add this if you want consistent hover styles for the light theme. It'd be better to have a variable for this as well.
```suggestion
}

.theme-light.maplibregl-ctrl button:not(:disabled):hover {
  background-color: #e2e8f0; /* Corresponds to slate-200 */
}
```